types/bitflags/bitflags16.ts

```diff
@@ -1,18 +1,19 @@
-import { SizedType, ViewableType } from "../types.ts";
+import { SizedType, TypeOptions, ViewableType } from "../types.ts";
 
 export class BitFlags16<
   T extends Record<string, number>,
   V extends Record<string, boolean> = { [K in keyof T]: boolean },
 > implements SizedType<V>, ViewableType<V> {
-  byteLength = 1;
+  byteLength = 2;
```

Comment:
Good catch! BitFlags should probably also align by their integer counterparts align (e.g. basically always their `byteLength`).